completor/create_output.py

```diff
@@ -363,12 +360,12 @@ def make_wsegicv(self, lateral: int) -> None:
     def make_wsegdar(self) -> None:
         """Print WSEGDAR to file."""
         if self.df_wsegdar.shape[0] > 0:
-            self.print_wsegdar += po.print_wsegdar(self.df_wsegdar, self.iwell + 1) + "\n"
+            self.print_wsegdar += po.print_wsegdar(self.df_wsegdar, self.well_number + 1) + "\n"
```

Comment:
Using `"".join` on a list of strings is way faster.'
https://stackoverflow.com/questions/2414667/python-string-class-like-stringbuilder-in-c